src/topp/ProteinQuantifier.cpp

```diff
@@ -710,7 +710,7 @@ class TOPPProteinQuantifier :
       IdXMLFile().load(in, proteins, peptides);
       for (Size i = 0; i < proteins.size(); ++i)
       {
-        files_[i].filename = proteins[i].getIdentifier();
+        files_[i].filename = proteins[i].getSearchEngine() + "_" + proteins[i].getDateTime().toString(Qt::ISODate);
```

Comment:
why not : `SE + date + UID` ?

Reply:
The output files of proteinquantifier looked like the UID does not belong there, so I treated the UID as an internal mapping and did not expose it